apps/meteor/tests/end-to-end/api/05-chat.js

```diff
@@ -785,44 +791,40 @@ describe('[Chat]', function () {
 			});
 
 			it('should have an iframe oembed with style max-width', (done) => {
-				setTimeout(() => {
-					request
-						.get(api('chat.getMessage'))
-						.set(credentials)
-						.query({
-							msgId: ytEmbedMsgId,
-						})
-						.expect('Content-Type', 'application/json')
-						.expect(200)
-						.expect((res) => {
-							expect(res.body).to.have.property('message').to.have.property('urls').to.be.an('array').that.is.not.empty;
+				request
```

Comment:
not sure about removing the `setTimeout` here.. the oembed action runs in a `afterSaveMessage` callback and it depends on how fast the CI can make the request and also how fast YT can respond to the request.. 

I have to admit though I'm shocked the test didn't fail.. but maybe removing the timeout can make it flaky